server/tests/test_closed_session.rs

```diff
@@ -12,6 +12,23 @@ enum Node {
     WithoutAlias,
 }
 
+fn tuples_vec_to_map<NODE: Eq + PartialEq + Hash + Clone>(
```

Comment:
Out of curiosity: why is this part of this PR while it doesn't seem to be used anywhere?